execution/consensus/misc/eip6110.go

```diff
@@ -56,9 +58,48 @@ type depositUnpacking struct {
 	Index                 []byte
 }
 
+var InvalidDepositLogErr = errors.New("invalid deposit log: unsupported data layout")
+
+func validateDepositLog(data []byte) error {
+	if len(data) != DepositLogLen {
+		return InvalidDepositLogErr
+	}
+	pubkey_offset := uint256.NewInt(0).SetBytes(data[0:32]).Uint64()
```

Comment:
The current logic will allow `pubkey_offset` with non-zero significant chunks (e.g. 160 + 2^128).

It's better to do smth. like
```
pubkey_offset := uint256.NewInt(0).SetBytes(data[0:32])
if pubkey_offset.CmpUint64(160) != 0 ||
```